core/build-readme.js

```diff
@@ -13,6 +12,9 @@ import qsm from 'qsm'
 
 import build from './build'
 
+const isEmpty = obj =>
+  [Object, Array].includes((obj || {}).constructor) && !Object.entries(obj || {}).length
```

Comment:
This one liner is really hard to read for me

Reply:
@wooorm You are right. Now it should be more readable :smile: